mapreduce/tests/test_mapreduce.py

```diff
@@ -91,12 +92,13 @@ def test_auth(aggregator, mocked_auth_request):
     """
     Test that we get all the metrics we're supposed to get
     """
+    instance = MR_AUTH_CONFIG['instances'][0]
 
     # Instantiate the check
-    mapreduce = MapReduceCheck("mapreduce", INIT_CONFIG, {})
+    mapreduce = MapReduceCheck('mapreduce', INIT_CONFIG, [instance])
```

Comment:
Question: What the point of passing `[instance]` as third argument if we calling later `mapreduce.check(instance)`. Legacy ?

---

By digging a bit into that:
Does this mean that all instances are using the same `RequestsWrapper` config (the first instance) ? Why ? :)
https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/base.py#L200-L201

Reply:
There will only ever be one instance

Reply:
You mean, for each "check instance" from config.yaml a new instance of `MapReduceCheck/AgentCheck` is created ?

Reply:
Yup, exactly! The signature is legacy from Agent 5